ImageManager/ImageManager.py

```diff
@@ -16,13 +17,31 @@
 
 from app import app
 
+import uuid
+from threading import Timer
+
 image = Blueprint('image', __name__)
 
 LOGGER = logging.getLogger('image-manager.' + __name__)
 LOGGER.addHandler(logging.StreamHandler())
-LOGGER.setLevel(logging.INFO)
+LOGGER.setLevel(logging.DEBUG)
+
+# global_imageid = 20
+
+
+UPLOAD_TIMEOUT = 5  # seconds
```

Comment:
I'd leave this a bit higher...
(or better yet, make this configurable - it depends on a number of factors that should be a bit hard to take into account at implementation time)